SDWebImage/SDImageCacheConfig.h

```diff
@@ -72,6 +72,12 @@
  */
 @property (assign, nonatomic) NSUInteger maxMemoryCount;
 
+/**
+ * The namespace prefix of cache. It's used to prefix the namespace you provide to the caches's initializer. You 'd better name it with reverse domain name notation and keep the final dot.
+ * Defautls to `com.hackemist.SDImageCache.`, which will prefix your namespace such as `default` to final `com.hackemist.SDImageCache.default`.
+ */
+@property (copy, nonatomic, nonnull) NSString *namespacePrefix;
```

Comment:
Can this be `nullable` ? Such as if you provide a nil value, we just ignore the prefix, the behavior is the same as `@""` ?

@AgentFeeble

Reply:
That makes sense, I can't see it causing any issues. And if you detect if it's nil, and replace it with an empty string (instead of asserting), it will cause less issues for users who are new to the library